log_injection/Kandhavel-S_DOC_BOOK/sp.py

```
@@ -0,0 +1,189 @@
+
+# Set up logging
+logging.basicConfig(level=logging.DEBUG)
+
+# Initialize the Flask app
+app = Flask(__name__)
+CORS(app)
+
+# Ensure the API key is set correctly
+api_key = "AIzaSyDFiEwUSBtEuIqfMVeVoTjWUIP4NGMyYFA"  # Replace with your actual API key
+if not api_key:
+    raise ValueError("No API_KEY found. Please set the `API_KEY` environment variable.")
```

Ruff (0.13.3)

Avoid specifying long messages outside the exception class

(TRY003)